python/cudf/cudf/core/column/column.py

```diff
@@ -386,6 +386,25 @@ def _get_mask_as_column(self) -> ColumnBase:
             self.base_mask, self.offset, self.offset + len(self)
         )
 
+    def _early_exit_eq_op(self, other):
```

Comment:
It seems like a better name would be `_create_all_false_column_with_nulls` or something to describe the nature of the output.

Also, should this be a method of a Column? I think it could be better as a free function in a utility namespace.

Reply:
Separated it out as a free function utility and also changed the name.